modules/upgrade-duration-mco.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-upgrade-duration.adoc
+
+:_content-type: CONCEPT
+[id="machine-config-operator_{context}"]
+= Machine Config Operator
+The Machine Config Operator (MCO) applies a new machine configuration to each control plane and worker node. During this process, the MCO performs the following sequential actions on each node of the cluster:
+
+. Cordon and drain all the nodes
+. Update the operating system (OS)
+. Reboot the nodes
+. Uncordon all nodes and schedule workloads on the node
+
+[NOTE]
+====
+When a node is cordoned, it is unavailable to schedule a workload.
+====
+
+The time to complete this process depends on several factors including the node and infrastructure configuration. This process might take 5 or more minutes to complete per node.
+
+In addition to MCO, you should consider the impact of following parameters:
+
+* The control plane node upgrade duration is predictable and oftentimes shorter than worker nodes, because the control plane workloads are tuned for graceful updates and quick drains.
```

Comment:
In our test, control plane node update seems more faster than worker node update.
```
# ./oc get mcp
NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
master   rendered-master-cc789c1d36213cf18743861f0ac9768d   False     True       False      3              1                   1                     0                      3h57m
worker   rendered-worker-0b67efd8082c03ad9d3220b1ce00ede2   True      False      False      2              2                   2                     0                      3h57m
```
From above, the control plane and worker node should start update at the same time, but 2 worker nodes has finished the update whereas only 1 control plane has finished update. Anything missing here?

Reply:
cc @wking to confirm.

Reply:
I wonder if it is because the worker nodes do not have any workload running on them.